pkgs/tools/misc/sbom-scorecard/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+}:
+
+buildGoModule rec {
+  pname = "sbom-scorecard";
+  version = "0.0.6";
+
+  src = fetchFromGitHub {
+    owner = "eBay";
+    repo = "sbom-scorecard";
+    rev = "${version}";
+    hash = "sha256-XE+c1IhtC5dHdVTQbtSqQky2Drwgd5ADzH8DdtJhYfA=";
```

Comment:
I'm not clear where these hashes come from. It may be worthwhile to put a url for someone to validate them.

Reply:
here is the useful thread for this purpose: _https://github.com/NixOS/nixpkgs/issues/191128_

Reply:
Gotcha. This is something already known to the nix community. All good from my perspective. :)